src/tracer.ts

```diff
@@ -1,11 +1,23 @@
 import { getTracer, startTracer, Tracer } from "@fwl/tracing";
 
-startTracer({
-  simpleCollector: {
-    serviceName: "connect-account",
-    url: "http://localhost:9411/api/v2/spans",
-  },
-});
+import { config } from "@src/config";
+
+const options = config.lightstepAccessToken
+  ? {
+      serviceName: config.tracerServiceName,
```

Comment:
you can get rid of this line but otherwise, 👍

Reply:
1a34baf
Ah you are right :).